.eslintrc

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "@vhx/eslint-config-vhx"
+  "extends": "@vhx/eslint-config-vhx",
+  "rules": {
+    "jsx-a11y/href-no-hash": 0
```

Comment:
I think this rule was added to @vhx/eslint-config-vhx, but it wasn't published to npm. For some reason, this causes a linting error on every file.